### ISSUE TITLE
ApolloError: Internal server error: locate_block_template()

### ISSUE THREAD

**Issue by krishaamer** [wpengine/headless-framework#356]
On a fresh install of Wordpress 5.8 Headless Frameworks fails with `ApolloError: Internal server error: locate_block_template(): Argument #3 ($templates) must be of type array, null given, called in wp-includes/template.php on line 66`

**Comment by blakewilson** [comment]
Hey @krishaamer thanks for opening this issue!

I am unable to replicate this. Does this happen on every page you visit or just one in particular?

Do you have a reproducible example I could take a look at?

**Comment by signorekai** [comment]
Commenting to confirm that I am also facing this issue, and it appears on all pages I visit.

**Comment by krishaamer** [comment]
@blakewilson Thanks. As I've been going through the issues it seems anything Apollo-related will be irelevant anyway and we should be looking at gqless instead?

**Comment by blakewilson** [comment]
@krishaamer Yes, this is true. We've since dropped the Apollo client in favor of GQless in `@faustjs`. The `@wpengine/headless` package will be phased out soon.

The latest up-to-date example can be cloned using:

```bash
npx create-next-app \
    -e https://github.com/wpengine/headless-framework/tree/canary \
    --example-path examples/next/getting-started \
    --use-npm \
    my-app
cd my-app
```

We are in the process of rewriting the docs to fit this new functionality with GQless, among other things, but they are not live yet. So apologies for the confusion there, this is our #1 priority right now.

**Comment by wjohnsto** [comment]
I believe this issue happens with version 0.5.5 of the WordPress plugin when using WordPress 5.8. We have since corrected the problem, and are working to release a new version of the plugin. For now you can download the `wpe-headless` folder [here](https://github.com/wpengine/headless-framework/tree/canary/plugins), then zip it up and manually install it in your WordPress site. However, we should have a new version of the plugin released soon to resolve this.

**Comment by blakewilson** [comment]
@wjohnsto Is right. This was an issue in version 0.5.5 of the plugin. We've just released version 0.5.6 with a fix. You can [download the latest version here.](https://wp-product-info-staging.wpesvc.net/v1/plugins/wpe-headless?download)

Please let us know if you continue to see issues with the latest version!
